src/components/hero/hero.scss

```diff
@@ -0,0 +1,109 @@
+@font-face {
+    font-family: "Josefin";
+    src: url("../font/JosefinSans-VariableFont_wght.ttf");
+}
+@font-face {
+    font-family: Lato;
+    src: url("../font/Lato-Black.ttf");
+}
+$font-family:Josefin;
+$font-family2:Lato;
```

Comment:
these 10 lines must be inside of index.scss
because they are global styles, and in this file you keep styles for your component only